amundsen_application/static/js/components/DashboardPage/RunState/index.tsx

```diff
@@ -0,0 +1,46 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as React from 'react';
+
+import { HIT_RUN_TEXT, MISSED_RUN_TEXT, LAST_RUN_SUCCEEDED } from './constants';
+
+import './styles.scss';
+
+export interface RunStateProps {
+  lastRunState: string;
+}
+
+const MissedState: React.FC = () => {
+  return (
+    <div className="missed">
+      <div className="missed-icon">
+        <div className="exclamation-top" />
+        <div className="exclamation-bottom" />
+      </div>
+      <span className="status-text body-2 text-primary">{MISSED_RUN_TEXT}</span>
```

Comment:
I would avoid to use text classes, and instead add the text styles to the 'status-text' class with:
```
@import 'typography-default';
...

.missed .status-text {
  @extend %text-body-w2;
}